libmachine/provision/ubuntu.go

```diff
@@ -49,11 +49,14 @@ func (provisioner *UbuntuProvisioner) Service(name string, action pkgaction.Serv
 func (provisioner *UbuntuProvisioner) Package(name string, action pkgaction.PackageAction) error {
 	var packageAction string
 
+	updateMetadata := true
```

Comment:
Is the idea that eventually this will be configurable / something the users can turn off?